.github/workflows/prebuild.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           cd pack
           python ../tools/package_tools/generate_build_config.py --backend 'ort' \
             --system linux --output config.yml --device cpu --build-sdk --build-sdk-monolithic \
-            --build-sdk-python --sdk-dynamic-net
+            --build-sdk-python --sdk-dynamic-net --onnxruntime-dir=$ONNXRUNTIME_GPU_DIR
```

Comment:
@irexyc Hi, is this change OK?

Reply:
Sorry, I didn't find the right way to do it after I failed to submit a PR, so I may have modified this file by mistake

Reply:
Hi @Luo-Z13 , we don't need to add this args. There exists a env variable of `ONNXRUNTIME_DIR` and wil be used by default. You can add a commit to remove the change or use `git revert` to undo that commit.

Reply:
> Hi @Luo-Z13 , we don't need to add this arg. There exists a env variable of `ONNXRUNTIME_DIR` and wil be used by default. You can add a commit to remove the change or use `git revert` to undo that commit.

Okay, I will change it soon after I fix lint error and update mmdet.yml as well.